sql/mozfun/norm/product_info/README.md

```diff
@@ -1,20 +1,20 @@
 The mappings are as follows:
 
-app_name         | normalized_os | product          | canonical_name              | 2019  | 2020
----------------- | ------------- | ---------------- | --------------------------- | ----- | ----
-Firefox          | *             | Firefox          | Firefox for Desktop         | true  | true
-Fenix            | Android       | Fenix            | Firefox for Android (Fenix) | true  | true
-Fennec           | Android       | Fennec           | Firefox for Android (Fennec)| true  | true
-Firefox Preview  | Android       | Firefox Preview  | Firefox Preview for Android | true  | true
-Fennec           | iOS           | Firefox iOS      | Firefox for iOS             | true  | true
-FirefoxForFireTV | Android       | Firefox Fire TV  | Firefox for Fire TV         | false | false
-FirefoxConnect   | Android       | Firefox Echo     | Firefox for Echo Show       | true  | true
-Zerda            | Android       | Firefox Lite     | Firefox Lite                | true  | true
-Zerda_cn         | Android       | Firefox Lite CN  | Firefox Lite (China)        | false | false
-Focus            | Android       | Focus Android    | Firefox Focus for Android   | true  | true
-Focus            | iOS           | Focus iOS        | Firefox Focus for iOS       | true  | true
-Klar             | Android       | Klar Android     | Firefox Klar for Android    | false | false
-Klar             | iOS           | Klar iOS         | Firefox Klar for iOS        | false | false
-Lockbox          | Android       | Lockwise Android | Lockwise for Android        | true  | true
-Lockbox          | iOS           | Lockwise iOS     | Lockwise for iOS            | true  | true
-FirefoxReality*  | Android       | Firefox Reality  | Firefox Reality             | false | false
+app_name         | normalized_os | looker_app_name  | product          | canonical_app_name              | 2019  | 2020  | 2021
```

Comment:
Adding this specifically as `looker_app_name` might be confusing. This should either be [`looker_namespace`](https://docs.google.com/document/d/1-0Krpe8wB3rZjTj9ud2ywP101d-FMZXpx7Cz_KQ-l8w), or simply `app_name` (per the [`repositories.yaml`](https://github.com/mozilla/probe-scraper/blob/master/repositories.yaml#L260)).

Reply:
Indeed. I thought we had to avoid `app_name` for compatibility, but it only appeared as an input parameter, which is renamable.

I've refactored this PR to make the input value `legacy_app_name` and the output now includes `app_name` rather than `looker_app_name`.